2013/01/27 21:33:34 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\sigma_local\protected\controllers\SiteController.php:221)
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 C:\xampp\htdocs\yii13\framework\web\CController.php(286): SiteController->runAction()
#2 C:\xampp\htdocs\yii13\framework\web\CController.php(265): SiteController->runActionWithFilters()
#3 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): SiteController->run()
#4 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 C:\xampp\htdocs\sigma_local\index.php(13): CWebApplication->run()
REQUEST_URI=/sigma_local/index.php?r=site/UpdateAjax&_=1359293613362
in C:\xampp\htdocs\sigma_local\protected\controllers\SiteController.php (221)
in C:\xampp\htdocs\sigma_local\protected\controllers\SiteController.php (185)
in C:\xampp\htdocs\sigma_local\index.php (13)
2013/01/27 21:33:36 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\sigma_local\protected\controllers\SiteController.php:221)
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 C:\xampp\htdocs\yii13\framework\web\CController.php(286): SiteController->runAction()
#2 C:\xampp\htdocs\yii13\framework\web\CController.php(265): SiteController->runActionWithFilters()
#3 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): SiteController->run()
#4 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 C:\xampp\htdocs\sigma_local\index.php(13): CWebApplication->run()
REQUEST_URI=/sigma_local/index.php?r=site/UpdateAjax&_=1359293615361
in C:\xampp\htdocs\sigma_local\protected\controllers\SiteController.php (221)
in C:\xampp\htdocs\sigma_local\protected\controllers\SiteController.php (185)
in C:\xampp\htdocs\sigma_local\index.php (13)
2013/01/28 15:07:05 [error] [exception.CHttpException.403] exception 'CHttpException' with message '您未被授权执行这个动作' in C:\xampp\htdocs\yii13\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), '???????????????...')
#1 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('project/admin')
#9 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projectGithubTest/index.php?r=project/admin
HTTP_REFERER=http://localhost/projectGithubTest/index.php?r=project/
---
2013/01/28 15:37:31 [error] [exception.CException] exception 'CException' with message '属性 "User.user_password" 未被定义.' in C:\xampp\htdocs\yii13\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\db\ar\CActiveRecord.php(143): CComponent->__get('user_password')
#1 C:\xampp\htdocs\projectGithubTest\protected\components\UserIdentity.php(28): CActiveRecord->__get('user_password')
#2 C:\xampp\htdocs\projectGithubTest\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii13\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii13\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii13\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\projectGithubTest\protected\controllers\SiteController.php(95): CModel->validate()
#7 C:\xampp\htdocs\yii13\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii13\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii13\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/projectGithubTest/index.php?r=site/login
HTTP_REFERER=http://localhost/projectGithubTest/index.php?r=site/login
---
2013/01/28 15:37:54 [error] [exception.CException] exception 'CException' with message 'User and its behaviors do not have a method or closure named "validatePassword".' in C:\xampp\htdocs\yii13\framework\base\CComponent.php:265
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\db\ar\CActiveRecord.php(225): CComponent->__call('validatePasswor...', Array)
#1 C:\xampp\htdocs\projectGithubTest\protected\components\UserIdentity.php(28): CActiveRecord->__call('validatePasswor...', Array)
#2 C:\xampp\htdocs\projectGithubTest\protected\components\UserIdentity.php(28): User->validatePassword('test1')
#3 C:\xampp\htdocs\projectGithubTest\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#4 C:\xampp\htdocs\yii13\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#5 C:\xampp\htdocs\yii13\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#6 C:\xampp\htdocs\yii13\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#7 C:\xampp\htdocs\projectGithubTest\protected\controllers\SiteController.php(95): CModel->validate()
#8 C:\xampp\htdocs\yii13\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 C:\xampp\htdocs\yii13\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii13\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('login')
#13 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#14 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/projectGithubTest/index.php?r=site/login
HTTP_REFERER=http://localhost/projectGithubTest/index.php?r=site/login
---
2013/01/28 15:48:34 [error] [php] include(UserOnline.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii13\framework\YiiBase.php:421)
Stack trace:
#0 C:\xampp\htdocs\projectGithubTest\protected\controllers\SiteController.php(99): spl_autoload_call()
#1 C:\xampp\htdocs\yii13\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#2 C:\xampp\htdocs\yii13\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\xampp\htdocs\yii13\framework\web\CController.php(286): SiteController->runAction()
#4 C:\xampp\htdocs\yii13\framework\web\CController.php(265): SiteController->runActionWithFilters()
#5 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): SiteController->run()
#6 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController()
#7 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\xampp\htdocs\projectGithubTest\index.php(13): CWebApplication->run()
REQUEST_URI=/projectGithubTest/index.php?r=site/login
in C:\xampp\htdocs\projectGithubTest\protected\controllers\SiteController.php (99)
in C:\xampp\htdocs\projectGithubTest\index.php (13)
2013/01/28 15:49:49 [error] [php] include(Message.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii13\framework\YiiBase.php:421)
Stack trace:
#0 C:\xampp\htdocs\projectGithubTest\protected\controllers\SiteController.php(255): spl_autoload_call()
#1 C:\xampp\htdocs\projectGithubTest\protected\controllers\SiteController.php(111): SiteController->addToTblChatcont()
#2 C:\xampp\htdocs\yii13\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#3 C:\xampp\htdocs\yii13\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 C:\xampp\htdocs\yii13\framework\web\CController.php(286): SiteController->runAction()
#5 C:\xampp\htdocs\yii13\framework\web\CController.php(265): SiteController->runActionWithFilters()
#6 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): SiteController->run()
#7 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\xampp\htdocs\projectGithubTest\index.php(13): CWebApplication->run()
REQUEST_URI=/projectGithubTest/index.php?r=site/login
in C:\xampp\htdocs\projectGithubTest\protected\controllers\SiteController.php (255)
in C:\xampp\htdocs\projectGithubTest\protected\controllers\SiteController.php (111)
in C:\xampp\htdocs\projectGithubTest\index.php (13)
2013/01/28 16:00:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message '无法解析请求 "issue"。' in C:\xampp\htdocs\yii13\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('issue')
#1 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projectGithubTest/index.php?r=issue
---
2013/01/28 23:58:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message '系统无法找到请求的 "list" 动作.' in C:\xampp\htdocs\yii13\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\CController.php(270): CController->missingAction('list')
#1 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('list')
#2 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('issue/list')
#3 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/projectGithubTest/index.php?r=issue/list
---
2013/01/28 23:58:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message '系统无法找到请求的 "list" 动作.' in C:\xampp\htdocs\yii13\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\CController.php(270): CController->missingAction('list')
#1 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('list')
#2 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('issue/list')
#3 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/projectGithubTest/index.php?r=issue/list
---
2013/01/29 00:03:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message '系统无法找到请求的 "list" 动作.' in C:\xampp\htdocs\yii13\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\CController.php(270): CController->missingAction('list')
#1 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('list')
#2 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('issue/list')
#3 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/projectGithubTest/index.php?r=issue/list
---
2013/01/29 00:03:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message '1111111111The requested project does not exist.' in C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php:186
Stack trace:
#0 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(206): IssueController->loadProject('5')
#1 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): IssueController->filterProjectContext(Object(CFilterChain))
#2 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#4 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#6 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#8 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('create')
#10 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('issue/create')
#11 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/projectGithubTest/index.php?r=issue/create&pid=5
---
2013/01/29 00:07:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message '1111111111The requested project does not exist.' in C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php:186
Stack trace:
#0 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(206): IssueController->loadProject(NULL)
#1 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): IssueController->filterProjectContext(Object(CFilterChain))
#2 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#4 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#6 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#8 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('create')
#10 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('issue/create')
#11 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/projectGithubTest/index.php?r=issue/create
---
2013/01/29 00:33:54 [error] [exception.CException] exception 'CException' with message '属性 "Issue._project" 未被定义.' in C:\xampp\htdocs\yii13\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\db\ar\CActiveRecord.php(143): CComponent->__get('_project')
#1 C:\xampp\htdocs\projectGithubTest\protected\models\Issue.php(150): CActiveRecord->__get('_project')
#2 C:\xampp\htdocs\projectGithubTest\protected\views\issue\_form.php(45): Issue->getProject()
#3 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii13\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\projectGithubTest\protected\views\issue\create.php(18): CController->renderPartial('_form', Array)
#7 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#8 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii13\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#10 C:\xampp\htdocs\yii13\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#11 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(83): CController->render('create', Array)
#12 C:\xampp\htdocs\yii13\framework\web\actions\CInlineAction.php(49): IssueController->actionCreate()
#13 C:\xampp\htdocs\yii13\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#14 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#15 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(211): CFilterChain->run()
#16 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): IssueController->filterProjectContext(Object(CFilterChain))
#17 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#19 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#23 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('create')
#25 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('issue/create')
#26 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#27 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/projectGithubTest/index.php?r=issue/create&pid=1
HTTP_REFERER=http://localhost/projectGithubTest/index.php?r=project/view&id=1
---
2013/01/29 00:34:26 [error] [exception.CException] exception 'CException' with message '属性 "Issue._project" 未被定义.' in C:\xampp\htdocs\yii13\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\db\ar\CActiveRecord.php(143): CComponent->__get('_project')
#1 C:\xampp\htdocs\projectGithubTest\protected\models\Issue.php(150): CActiveRecord->__get('_project')
#2 C:\xampp\htdocs\projectGithubTest\protected\views\issue\_form.php(45): Issue->getProject()
#3 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii13\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\projectGithubTest\protected\views\issue\create.php(18): CController->renderPartial('_form', Array)
#7 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#8 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii13\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#10 C:\xampp\htdocs\yii13\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#11 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(83): CController->render('create', Array)
#12 C:\xampp\htdocs\yii13\framework\web\actions\CInlineAction.php(49): IssueController->actionCreate()
#13 C:\xampp\htdocs\yii13\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#14 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#15 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(211): CFilterChain->run()
#16 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): IssueController->filterProjectContext(Object(CFilterChain))
#17 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#19 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#23 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('create')
#25 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('issue/create')
#26 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#27 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/projectGithubTest/index.php?r=issue/create&pid=1
HTTP_REFERER=http://localhost/projectGithubTest/index.php?r=project/view&id=1
---
2013/01/29 00:36:00 [error] [exception.CException] exception 'CException' with message 'Project and its behaviors do not have a method or closure named "getUserOptions".' in C:\xampp\htdocs\yii13\framework\base\CComponent.php:265
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\db\ar\CActiveRecord.php(225): CComponent->__call('getUserOptions', Array)
#1 C:\xampp\htdocs\projectGithubTest\protected\views\issue\_form.php(45): CActiveRecord->__call('getUserOptions', Array)
#2 C:\xampp\htdocs\projectGithubTest\protected\views\issue\_form.php(45): Project->getUserOptions()
#3 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii13\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\projectGithubTest\protected\views\issue\create.php(18): CController->renderPartial('_form', Array)
#7 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#8 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii13\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#10 C:\xampp\htdocs\yii13\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#11 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(83): CController->render('create', Array)
#12 C:\xampp\htdocs\yii13\framework\web\actions\CInlineAction.php(49): IssueController->actionCreate()
#13 C:\xampp\htdocs\yii13\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#14 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#15 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(211): CFilterChain->run()
#16 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): IssueController->filterProjectContext(Object(CFilterChain))
#17 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#19 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#23 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('create')
#25 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('issue/create')
#26 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#27 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/projectGithubTest/index.php?r=issue/create&pid=1
HTTP_REFERER=http://localhost/projectGithubTest/index.php?r=project/view&id=1
---
2013/01/29 00:39:57 [error] [exception.CException] exception 'CException' with message 'Project and its behaviors do not have a method or closure named "getUserOptions".' in C:\xampp\htdocs\yii13\framework\base\CComponent.php:265
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\db\ar\CActiveRecord.php(225): CComponent->__call('getUserOptions', Array)
#1 C:\xampp\htdocs\projectGithubTest\protected\views\issue\_form.php(45): CActiveRecord->__call('getUserOptions', Array)
#2 C:\xampp\htdocs\projectGithubTest\protected\views\issue\_form.php(45): Project->getUserOptions()
#3 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii13\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\projectGithubTest\protected\views\issue\create.php(18): CController->renderPartial('_form', Array)
#7 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#8 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii13\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#10 C:\xampp\htdocs\yii13\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#11 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(83): CController->render('create', Array)
#12 C:\xampp\htdocs\yii13\framework\web\actions\CInlineAction.php(49): IssueController->actionCreate()
#13 C:\xampp\htdocs\yii13\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#14 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#15 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(211): CFilterChain->run()
#16 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): IssueController->filterProjectContext(Object(CFilterChain))
#17 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#19 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#23 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('create')
#25 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('issue/create')
#26 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#27 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/projectGithubTest/index.php?r=issue/create&pid=1
HTTP_REFERER=http://localhost/projectGithubTest/index.php?r=project/view&id=1
---
2013/01/29 00:40:00 [error] [exception.CException] exception 'CException' with message 'Project and its behaviors do not have a method or closure named "getUserOptions".' in C:\xampp\htdocs\yii13\framework\base\CComponent.php:265
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\db\ar\CActiveRecord.php(225): CComponent->__call('getUserOptions', Array)
#1 C:\xampp\htdocs\projectGithubTest\protected\views\issue\_form.php(45): CActiveRecord->__call('getUserOptions', Array)
#2 C:\xampp\htdocs\projectGithubTest\protected\views\issue\_form.php(45): Project->getUserOptions()
#3 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii13\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\projectGithubTest\protected\views\issue\create.php(18): CController->renderPartial('_form', Array)
#7 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#8 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii13\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#10 C:\xampp\htdocs\yii13\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#11 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(83): CController->render('create', Array)
#12 C:\xampp\htdocs\yii13\framework\web\actions\CInlineAction.php(49): IssueController->actionCreate()
#13 C:\xampp\htdocs\yii13\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#14 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#15 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(211): CFilterChain->run()
#16 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): IssueController->filterProjectContext(Object(CFilterChain))
#17 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#19 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#23 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('create')
#25 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('issue/create')
#26 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#27 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/projectGithubTest/index.php?r=issue/create&pid=1
HTTP_REFERER=http://localhost/projectGithubTest/index.php?r=project/view&id=1
---
2013/01/29 00:40:07 [error] [exception.CException] exception 'CException' with message 'Project and its behaviors do not have a method or closure named "getUserOptions".' in C:\xampp\htdocs\yii13\framework\base\CComponent.php:265
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\db\ar\CActiveRecord.php(225): CComponent->__call('getUserOptions', Array)
#1 C:\xampp\htdocs\projectGithubTest\protected\views\issue\_form.php(45): CActiveRecord->__call('getUserOptions', Array)
#2 C:\xampp\htdocs\projectGithubTest\protected\views\issue\_form.php(45): Project->getUserOptions()
#3 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii13\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\projectGithubTest\protected\views\issue\create.php(18): CController->renderPartial('_form', Array)
#7 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#8 C:\xampp\htdocs\yii13\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii13\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#10 C:\xampp\htdocs\yii13\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#11 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(83): CController->render('create', Array)
#12 C:\xampp\htdocs\yii13\framework\web\actions\CInlineAction.php(49): IssueController->actionCreate()
#13 C:\xampp\htdocs\yii13\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#14 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#15 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(211): CFilterChain->run()
#16 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): IssueController->filterProjectContext(Object(CFilterChain))
#17 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#19 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#23 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('create')
#25 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('issue/create')
#26 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#27 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/projectGithubTest/index.php?r=issue/create&pid=1
HTTP_REFERER=http://localhost/projectGithubTest/index.php?r=project/view&id=1
---
2013/01/29 00:59:10 [error] [exception.CHttpException.400] exception 'CHttpException' with message '您的请求无效。' in C:\xampp\htdocs\yii13\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('project/view')
#10 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/projectGithubTest/index.php?r=project/view
---
2013/01/29 00:59:12 [error] [exception.CHttpException.400] exception 'CHttpException' with message '您的请求无效。' in C:\xampp\htdocs\yii13\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('project/view')
#10 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/projectGithubTest/index.php?r=project/view
---
2013/01/29 00:59:13 [error] [exception.CHttpException.400] exception 'CHttpException' with message '您的请求无效。' in C:\xampp\htdocs\yii13\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('project/view')
#10 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/projectGithubTest/index.php?r=project/view
---
2013/01/29 00:59:14 [error] [exception.CHttpException.400] exception 'CHttpException' with message '您的请求无效。' in C:\xampp\htdocs\yii13\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('project/view')
#10 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/projectGithubTest/index.php?r=project/view
---
2013/01/29 00:59:15 [error] [exception.CHttpException.400] exception 'CHttpException' with message '您的请求无效。' in C:\xampp\htdocs\yii13\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('project/view')
#10 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/projectGithubTest/index.php?r=project/view
---
2013/01/29 00:59:16 [error] [exception.CHttpException.400] exception 'CHttpException' with message '您的请求无效。' in C:\xampp\htdocs\yii13\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('project/view')
#10 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/projectGithubTest/index.php?r=project/view
---
2013/01/29 00:59:17 [error] [exception.CHttpException.400] exception 'CHttpException' with message '您的请求无效。' in C:\xampp\htdocs\yii13\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('project/view')
#10 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/projectGithubTest/index.php?r=project/view
---
2013/01/29 00:59:18 [error] [exception.CHttpException.400] exception 'CHttpException' with message '您的请求无效。' in C:\xampp\htdocs\yii13\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('project/view')
#10 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/projectGithubTest/index.php?r=project/view
---
2013/01/29 00:59:18 [error] [exception.CHttpException.400] exception 'CHttpException' with message '您的请求无效。' in C:\xampp\htdocs\yii13\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii13\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('project/view')
#10 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/projectGithubTest/index.php?r=project/view
---
2013/01/29 01:07:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message '1111111111The requested project does not exist.' in C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php:188
Stack trace:
#0 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(208): IssueController->loadProject(NULL)
#1 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): IssueController->filterProjectContext(Object(CFilterChain))
#2 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#4 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#6 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#8 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('create')
#10 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('issue/create')
#11 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/projectGithubTest/index.php?r=issue/create
HTTP_REFERER=http://localhost/projectGithubTest/index.php?r=issue
---
2013/01/29 01:07:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message '1111111111The requested project does not exist.' in C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php:188
Stack trace:
#0 C:\xampp\htdocs\projectGithubTest\protected\controllers\IssueController.php(208): IssueController->loadProject(NULL)
#1 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): IssueController->filterProjectContext(Object(CFilterChain))
#2 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii13\framework\web\filters\CFilter.php(40): CFilterChain->run()
#4 C:\xampp\htdocs\yii13\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii13\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#6 C:\xampp\htdocs\yii13\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii13\framework\web\CController.php(291): CFilterChain->run()
#8 C:\xampp\htdocs\yii13\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(282): CController->run('create')
#10 C:\xampp\htdocs\yii13\framework\web\CWebApplication.php(141): CWebApplication->runController('issue/create')
#11 C:\xampp\htdocs\yii13\framework\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\xampp\htdocs\projectGithubTest\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/projectGithubTest/index.php?r=issue/create
HTTP_REFERER=http://localhost/projectGithubTest/index.php?r=issue/view&id=2
---
